tests/test_app.py

```diff
@@ -3,12 +3,14 @@
 # This file is part of Invenio.
 # Copyright (C) 2015-2022 CERN.
 # Copyright (C) 2022 RERO.
+# Copyright (C) 2025 Graz University of Technology.
 #
 # Invenio is free software; you can redistribute it and/or modify it
 # under the terms of the MIT License; see LICENSE file for more details.
 
 """Test basic application."""
 
+import importlib
```

Comment:
We should import it this way:

```suggestion
import importlib.metadata
```

By importing it this way, the code that checks the version (`importlib.metadata.version("click")` below) will work on Python 3.9 and above. Otherwise it won't work on Python 3.9, because `importlib.metadata` was only made no longer provisional in Python 3.10 .